### PR TITLE
Add coin token tracking

### PR DESCRIPTION
**Pull request by dbergan**
And some minor tweaks to singular/plural wording (e.g. "1 VP chips" --> "1 VP chip")

### CODE REVIEW

File: web/static/logprettifier.html [hunk 1]
```diff
@@ -410,41 +418,47 @@
 function vp_in_deck(deck) {
     var points = 0;
     for (var card in deck) if(vpoint[card]) {
-	points += deck[card] * vpoint[card](deck);
```
Comment (amalloy) [inline web/static/logprettifier.html]: These extraneous whitespace changes are a little tacky to include; can you restrict this diff to only relevant changes?

**Comment by amalloy** [comment]
I really like this feature, if it works. I'm not in charge, but I hope @aiannacc takes it.

File: web/static/logprettifier.html (hunk 1 above)
Comment (dbergan) [inline web/static/logprettifier.html]: Yeah, my code editor program made those changes. Can you explain how to restrict the diff? I'm pretty new to github.

**Comment by dbergan** [comment]
Examples of the feature in action. 

http://aerolegra.com/logprettifier?20150215/log.50e86751e4b0d990e88ce76f.1424060981439.txt

http://aerolegra.com/logprettifier?20150215/log.50e86751e4b0d990e88ce76f.1424060981439.txt

**Pull request closed without merging**